sp800-63b/sec5_authenticators.md

```diff
@@ -399,4 +399,8 @@ The latter class of verifier impersonation resistant protocols relies on access
 
 In contrast, authenticators that involve the manual entry of an authenticator output, such as out of band and one-time password authenticators, SHALL NOT be considered verifier impersonation resistant because they assume the vigilance of the claimant to determine that they are communicating with the intended verifier.
 
+#### <a name="csp-verifier"></a>5.2.6. Verifier-CSP commmunications
 
+In situations where the verifier and CSP are separate entities, communications between the verifier and CSP SHALL occur through a mutually-authenticated secure channel (such as a client-authenticated TLS connection) using approved cryptography.
```

Comment:
This could be construed as federation/assertions, so we either need to be clear as to why it's not, or if it is, point to -C.